examples/context/index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
```

Comment:
Yeah, this was just the branch I was working on for testing my stuff visually and so all this code would need cleaning up a lot before actually going in. Seb opened up a pr for discussions sake :)

It's great to see all the discussion which has been going on though! I've been busy and away over the past week so haven't been able to get involved too much. Hopefully will have a read through everything later!